airflow/www/views.py

```
@@ -2288,6 +2291,38 @@ class UserModelView(wwwutils.SuperUserMixin, AirflowModelView):
     column_default_sort = 'username'
 
 
+class VersionView(wwwutils.SuperUserMixin, BaseView):
+    @expose('/')
+    def version(self):
+        # Look at the version from setup.py
+        airflow_version = None
+        try:
+            airflow_version = pkg_resources.require("airflow")[0].version
+        except :
+            pass
```

Nit: Could use a log.debug here rather than swallowing the error